domain/credentials/service.go

```diff
@@ -46,28 +46,27 @@ func (s Service) ManageNutsOrgCredential(customer domain.Customer, shouldHaveCre
 		if customer.City == nil {
 			return fmt.Errorf("customer.City must be set for issuing a credential")
 		}
-		issue := true
-		for _, credential := range credentials {
-			if credential.Organization.Name == customer.Name && credential.Organization.City == *customer.City {
-				// Customer already has credential with given name/ & city, so we don't need to issue one.
-				issue = false
-				break
+
+		if len(credentials) > 0 {
+			if err := s.RevokeCredentials(credentials); err != nil {
+				return fmt.Errorf("unable to revoke NutsOrgCredentials for customer %d: %w", customer.Id, err)
 			}
 		}
-		if issue {
-			err = s.issueNutsOrgCredential(customer)
+
+		if err := s.issueNutsOrgCredential(customer); err != nil {
+			return fmt.Errorf("unable to manage NutsOrgCredential for customer %d: %w", customer.Id, err)
 		}
 	} else {
```

Comment:
It looks you are always revoking in both the `shouldHaveCredential` is true or false. Perhaps take it out of the conditional and remove the else?